rest-api/tools/import_organizations.py

```diff
@@ -122,6 +124,9 @@ def __init__(self):
     self.organization_dao = OrganizationDao()
     args = parser.parse_args()
     self.geocode_flag = args.geocode_flag
+    self.errors = list()
```

Comment:
I'd move self.errors into the base class; and add to it in all importers wherever we currently log a warning.

Reply:
done